src/EasyAbp.BookingService.Domain/EasyAbp/BookingService/Assets/AssetManager.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using EasyAbp.BookingService.AssetCategories;
+using EasyAbp.BookingService.AssetDefinitions;
+using EasyAbp.BookingService.AssetSchedules;
+using JetBrains.Annotations;
+using Microsoft.Extensions.Options;
+using Volo.Abp;
+using Volo.Abp.Domain.Services;
+using Volo.Abp.Uow;
+
+namespace EasyAbp.BookingService.Assets;
+
+public class AssetManager : DomainService, IUnitOfWorkEnabled
```

Comment:
I think the `IUnitOfWorkEnabled` is not a good way.
1. Not all methods in the `AssetManager` require a UOW.
2. The `IUnitOfWorkEnabled` interface doesn't decide whether it needs a transaction.

Even if we use `IUnitOfWorkEnabled`, we should make methods `virtual` since it is good for the ABP module extension.